pyhomematic/_hm.py

```diff
@@ -669,14 +680,21 @@ def proxyInit(self):
             LOG.debug("ServerThread.proxyInit: init('http://%s:%i', '%s')" %
                       (callbackip, callbackport, interface_id))
             try:
+                # at least for home ip, init ccu is not working. Read list devices before "init proxy"
+                if interface_id == "homeassistant-HMIP":
```

Comment:
This will only work if the user has named the configuration exactly `HMIP` in Home Assistant. I think it would make more sense to check if the port of the interface is `2010` or (I believe for SSL) `32010`.

Reply:
in deed, this is very specific. It resulted from the observation, that this listdevice method does not work for most of the other interfaces. I can for sure use the port for selection and resubmit.